dist/css/ui-components.css

```diff
@@ -1,2 +1,252 @@
+.toolbar-pf .form-group .btn-group {
+  /* overrides _toolbar.scss to account for intermediate Angular markup */
+  margin-left: 5px; }
+
+.form-group.miq-toolbar-group {
+  /* styling for "Custom Html Toolbar Menu" example (used on classic-ui topology screen) */
+  display: inline-block; }
+  .form-group.miq-toolbar-group .miq-custom-html form {
+    display: inline-block; }
+  .form-group.miq-toolbar-group .miq-custom-html .form-group {
+    margin-bottom: 0 !important;
+    border-right: 0; }
+    .form-group.miq-toolbar-group .miq-custom-html .form-group:last-child {
+      padding-right: 0px; }
+
+/* End Patternfly Toolbar overrides */
+/* Begin Patternfly Tab overrides used in the Dialog Editor */
+.dialog-editor-tab-list {
+  margin-bottom: 20px; }
+
+.delete-tab {
+  opacity: .3;
+  visibility: visible; }
+
+.select-tab:hover .delete-tab {
+  opacity: 1;
+  visibility: visible; }
+
+/* End Patternfly Tab overrides */
+/* Begin Patternfly Table View overrides */
+.miq-data-tables-header {
+  text-align: left; }
+  .miq-data-tables-header .pull-right {
+    padding-right: 20px; }
+    .miq-data-tables-header .pull-right div {
+      display: inline-block;
+      margin-right: 10px; }
+
+table.miq-table-with-footer {
+  /* not currently used */
+  margin-bottom: 0px; }
+
+.miq-table thead th {
+  color: #0099d3;
+  cursor: pointer; }
+
+.dataTables_footer span {
+  margin-left: 10px; }
+
+/* End Patternfly Table View overrides */
+/* Begin Grid View (Quad Icon) & Tile View Styling */
+.miq-tile-section .card .card-content {
+  padding-right: 5px; }
+
+.miq-tile-section div.miq-paging {
+  text-align: center; }
+  .miq-tile-section div.miq-paging ul {
+    margin-top: 0; }
+
+.miq-tile-view .card-view-pf {
+  /* Styling for Cards in Tile & Grid Views  */ }
+  .miq-tile-view .card-view-pf .card {
+    padding: 0 !important;
+    cursor: pointer; }
+    .miq-tile-view .card-view-pf .card .card-content > div {
+      padding-top: 10px; }
+    .miq-tile-view .card-view-pf .card .card-check-box {
+      left: 0 !important;
+      top: 12px !important; }
+
+.miq-tile-with-body .card-view-pf {
+  /* Styling for Cards in Tile View */ }
+  .miq-tile-with-body .card-view-pf .card {
+    width: 375px;
+    height: 175px; }
+    .miq-tile-with-body .card-view-pf .card .card-content {
+      overflow: hidden; }
+    .miq-tile-with-body .card-view-pf .card .miq-icon-section {
+      padding-right: 0; }
+    .miq-tile-with-body .card-view-pf .card .miq-info-section {
+      padding-left: 0; }
+      .miq-tile-with-body .card-view-pf .card .miq-info-section dl.dl-horizontal.tile dt {
+        margin-left: 10px; }
+      .miq-tile-with-body .card-view-pf .card .miq-info-section dl.dl-horizontal.tile dd {
+        margin-left: 120px;
+        text-align: left; }
+
+.miq-row-margin-only-top {
+  /* Adds a row after the title in Tile View Card */
+  margin-top: 15px;
+  margin-left: 0;
+  margin-right: 0; }
+
+.miq-small-tile .card-view-pf {
```

Comment:
They are not, this is a generated file which changed from single-line to multiline.

(@skateman you may want to run `webpack -p` to recompile these..)